frontend/www/js/omegaup/components/course/CardsList.vue

```diff
@@ -13,14 +13,47 @@
       <template v-for="(typeCourses, accessMode) in courses">
         <div v-if="typeCourses.activeTab !== ''" :key="accessMode" class="row">
           <div class="col-lg-5 p-3 d-flex" :class="accessMode">
-            <h3 class="flex-grow-1">{{ getDescription(accessMode) }}</h3>
+            <h3 class="flex-grow-1 text-white">
+              {{ getDescription(accessMode) }}
+            </h3>
+            <a
+              role="button"
+              class="text-white"
+              data-toggle="modal"
+              :data-target="`.${accessMode}-modal`"
+            >
+              <font-awesome-icon
+                icon="info-circle"
+                :title="T[`${accessMode}CourseInformationDescription`]"
+              />
+            </a>
             <div
-              class="d-inline-block"
-              tabindex="0"
-              data-toggle="tooltip"
-              :title="T[`${accessMode}CourseInformationDescription`]"
+              class="modal text-black"
+              :class="`${accessMode}-modal`"
+              tabindex="-1"
             >
-              <font-awesome-icon icon="info-circle" />
+              <div class="modal-dialog modal-dialog-centered">
+                <div class="modal-content">
+                  <div class="modal-header">
+                    <h5 class="modal-title">
+                      {{ getDescription(accessMode) }}
+                    </h5>
+                    <button
+                      type="button"
+                      class="close"
+                      data-dismiss="modal"
+                      aria-label="Close"
+                    >
+                      <span aria-hidden="true">&times;</span>
+                    </button>
+                  </div>
+                  <div class="modal-body">
+                    <omegaup-markdown
+                      :markdown="T[`${accessMode}CoursesModal`]"
+                    ></omegaup-markdown>
```

Comment:
Para que no se queje el linter de que no estás utilizando esas cadenas de traducción se me ocurre que agregues un método donde le puedas mandar el `accessMode` como variable:
```suggestion
                    <omegaup-markdown
                      :markdown="courseModalByType(acessMode)"
                    ></omegaup-markdown>
```
Y este método te regrese la cadena que esperas, tal cual como aparece en el archivo de traducción:
```typescript
courseModalByType(accessMode: string): string {
  if (accessMode === 'public') {
    return T.publicCoursesModal;
  }
  return T.studentCoursesModal;
}

Reply:
Ahhh, tienes razón, gracias!